lambdas/refresh.js

```diff
@@ -0,0 +1,155 @@
+'use strict'
+
+const https = require('https')
+const querystring = require('querystring')
+const { SSM, CognitoIdentity, CognitoSync } = require('aws-sdk')
+
+const ssm = new SSM()
+
+exports.handler = async (event, context, callback) => {
```

Comment:
```suggestion
exports.handler = async (event, context) => {
```

With Node.js 8 support we use the [async model](https://aws.amazon.com/blogs/compute/node-js-8-10-runtime-now-available-in-aws-lambda/) like in authorize.js.